generator/sig_index.tmpl

```diff
@@ -0,0 +1,27 @@
+{{- template "header" }}
+# {{.Name}} SIG
+
+{{ .MissionStatement }}
+## Meetings
+{{- range .Meetings }}
+* [{{.Day}}s at {{.UTC}} UTC]({{$.MeetingURL}}) ({{.Frequency}})
```

Comment:
Suggestion: time zones are hard.  Suggestion: perhaps link the time to some handy service like http://www.thetimezoneconverter.com/event_link.html.  I just pulled that one off of Google but it looks nice enough.